rethinkdb/README.md

```diff
@@ -0,0 +1,92 @@
+# Supported tags and respective `Dockerfile` links
+
+- [`1.15.1` (*1.15.1/Dockerfile*)](https://github.com/stuartpb/rethinkdb-dockerfiles/blob/8990b61b5881039e15049f3b73191e37ba640cff/1.15.1/Dockerfile)
+- [`1.15.2` (*1.15.2/Dockerfile*)](https://github.com/stuartpb/rethinkdb-dockerfiles/blob/8990b61b5881039e15049f3b73191e37ba640cff/1.15.2/Dockerfile)
+- [`1.15.3`, `1.15`, `1`, `latest` (*1.15.3/Dockerfile*)](https://github.com/stuartpb/rethinkdb-dockerfiles/blob/8990b61b5881039e15049f3b73191e37ba640cff/1.15.3/Dockerfile)
+
+For more information about this image and its history, please see the [relevant
+manifest file
+(`library/rethinkdb`)](https://github.com/docker-library/official-images/blob/master/library/rethinkdb)
+in the [`docker-library/official-images` GitHub
+repo](https://github.com/docker-library/official-images).
+
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/rethinkdb/logo.png)
+
+# How to use this image
+
+## start an instance allowing connections on all interfaces
+
+```bash
+docker run --name some-rethink -d rethinkdb --bind all
+```
+
+## connect to it from an application ...
+
+```bash
+docker run --name some-app --link some-rethink:rdb -d application-that-uses-rdb
+```
+
+## ... via the web admin interface ...
+
+```bash
+# browsing on the same machine:
+xdg-open \
+  "http://$(docker inspect --format '{{ .NetworkSettings.IPAddress }}'):8080"
+
+# exposing and connecting remotely:
+docker run -d --link some-rethink:rdb -p 8080:8080 svendowideit/ambassador
+echo "click me: http://$(hostname):8080"
+```
+
+## ... or via the command-line interface
+
+```bash
+docker run -it --link some-rethink:rdb --rm rethinkdb sh -c 'exec \
+  rethinkdb admin --join "$RDB_PORT_29015_TCP_ADDR:$RDB_PORT_29015_TCP_PORT"'
+```
```

Comment:
This will need to be either replaced or altogether removed, as 1.16.0 has removed the `rethinkdb admin` CLI. (Presumably, the best way to administer RethinkDB from the command line now is using the REPL of a language with an official RethinkDB driver.)